TeamCode/src/main/java/org/firstinspires/ftc/teamcode/commands/Com_Shooter.java

```diff
@@ -0,0 +1,24 @@
+package org.firstinspires.ftc.teamcode.commands;
+
+import com.arcrobotics.ftclib.command.CommandBase;
+
+import org.firstinspires.ftc.teamcode.subsystems.ShooterSubsystem;
+
+public class Com_Shooter extends CommandBase {
+
+    private ShooterSubsystem shooterSubsystem;
+
+    public Com_Shooter(ShooterSubsystem subsystem){
+        shooterSubsystem = subsystem;
+        addRequirements(shooterSubsystem);
+    }
+
+    public void execute(){
+        shooterSubsystem.flickReset();
```

Comment:
Move this to an `initialize()` and add an `isFinished()` that returns `!shooterSubsystem.flickerRunning()` which returns `flicker.running()`. If you don't move this line to `initialize()`, it will constantly reset and never end (and also never work).